views/frontend/post.php

```diff
@@ -1,71 +1,68 @@
-<?php
-    $post = $model_article->get_article_blog();
-    if($post == false){
-        header("Location:/page-erreur");
-    }else{
-?>
-    
-    <div class="row center">
+<div class="row center">
         <div class=" row ">
-            <img class="responsive-img" src="/public/img/posts/<?= $post->image ?>" alt="<?= $post->title ?>"/>
+            <img class="responsive-img" src="/public/img/posts/<?= $post->getImage() ?>" alt="<?= $post->getTitle() ?>"/>
         </div>
     </div>
     
     <div class="container">
-            <h1><?= $post->title ?></h1>
-            <h6>Par <?= $post->name ?> le <?= date("d/m/Y à H:i", strtotime($post->date)) ?></h6>
-            <p><?= nl2br($post->content); ?></p>
+        <h1><?= $post->getTitle() ?></h1>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)